### ISSUE TITLE
[SCREENREADER]: Set focus on the update message, then back to Edit when user updates their profile mailing address

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6472]
## Description

**⚠️Should**

Focus should be set on the update message so screen reader users understand the mailing address is being written and saved, then moved back to the Edit button when the message drops out and the new address is added to the page. Screenshots attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/5865
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/6470

## Point of Contact

**VFS Point of Contact:** _Jennifer, Trevor can consult too_

## Acceptance Criteria

- [ ] As a screen reader user, I want to hear the update message read aloud, then "Edit your mailing address, button" when the new address is written to the page.

## Environment

* Windows 10
* IE11
* JAWS

## Steps to Recreate

1. Log into test user 56's `/profile`
2. Click the `Edit` button for mailing address
3. Enter a valid, overridable address
4. Press the `Update` button
5. Confirm your address in the modal, and press `Update`
6. Verify focus is on the `Edit` button, "falls away" while the update message is on screen, and goes back to the `Edit` button when the new address is added to the page.

## Possible Fixes (optional)
Use the React `focus` helper to set focus on the update message, then the Edit button again.

## WCAG or Vendor Guidance (optional)
* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2020-03-04 at 11 07 20 AM](https://user-images.githubusercontent.com/934879/75905428-72630f80-5e0a-11ea-9907-ba8b6a66cbdd.png)

---
![Screen Shot 2020-03-04 at 11 07 00 AM](https://user-images.githubusercontent.com/934879/75905438-78f18700-5e0a-11ea-90e0-074887340275.png)

**Comment by Samara-Strauss** [comment]
@bradl3yC @mshea0606 let's prioritize this for next sprint in favor of the other launch-required to-dos we have.

**Comment by bradl3yC** [comment]
@jenstrickland this is ready for validation

**Comment by jenstrickland** [comment]
Validated that this is fixed.

- Focus goes to the message notifying user the address is being updated
- Then, goes to the Edit button when update is complete

Great job! Thanks!
